<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="LblState">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QComboBox" name="CmbAudioInputs"/>
        </item>
        <item>
         <widget class="QComboBox" name="CmbAudioOutputs"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="LblMyID">
          <property name="text">
           <string>My ID [0 Disable] :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="SpbMyID">
          <property name="maximum">
           <number>254</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="LblSendToID">
          <property name="text">
           <string>Send To ID [255 Broadcast] :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="SpbSendToID">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="BtnSendData">
          <property name="text">
           <string>&amp;Send</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BtnExit">
          <property name="text">
           <string>&amp;Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>643</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
